examples/sdk/sdk_example.py

```diff
@@ -6,11 +6,11 @@
 # set your secrets
 secrets_confg = aztk.spark.models.SecretsConfiguration(
     service_principal=aztk.models.ServicePrincipalConfiguration(
-        tenant_id = "<org>.onmicrosoft.com",
-        client_id = "",
-        credential = "",
-        batch_account_resource_id = "",
-        storage_account_resource_id = "",
+        tenant_id="<org>.onmicrosoft.com",
```

Comment:
Is this what a tenant_id typically looks like? Mine look like UUIDs.

Reply:
yeah you can use the uuid or also the name, I tested that last week and it seems to work. This just look more friendly and its easier to find on the portal.